cooc.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
```

Comment:
What is this file? Is it the old implementation? this file should be inside src/test/resources I think.